README.md

```diff
@@ -250,6 +269,43 @@ the query result like below:
 }
 ```
 
+### Pub message and retrieve ZKP by `ioctl`
+
+```sh
+ioctl ws message send --project-id ${project id} --project-version ${project version} --data ${message content}
```

Comment:
change all wsctl and use ioctl to replease ?

Reply:
remove `wsctl` after all commands migrating to `ioctl`?